skins/oasis/css/modules/ContributeModule.scss

```diff
@@ -0,0 +1,46 @@
+@import 'skins/shared/mixins/flexbox';
+
+.contribute-module {
```

Comment:
Still trying to figure out what's causing this one. It's worth noting however it starts behaving weirdly again if there's >1m pages on a wiki.